mobile/src/main/java/org/openhab/habdroid/core/ConnectivityChangeReceiver.java

```diff
@@ -16,7 +16,7 @@
 import android.util.Log;
 
 public class ConnectivityChangeReceiver extends BroadcastReceiver {
-    private static final String TAG = "ConnectivityChangeReceiver";
+    private static final String TAG = ConnectivityChangeReceiver.class.getSimpleName();
```

Comment:
I think so. The lint warnings about the TAG being too long will go away and if you refactor the class name, the tag will be updated too.